assignment_w48.cpp

```diff
@@ -0,0 +1,62 @@
+#include <iostream>
+#include <cstdlib>
+#include <algorithm>
+#include <fstream>
+#include <vector>
+#include <string>
+using namespace std;
+
+/*  create a deck of cards that may be sorted using either their
+value (e.g., all aces, then all twos, then all threes ...) or suite-value combination.
+Finally make it possible to shuffle the deck. */
+
+// a card must be represented as a type with a value . eg: 7 of spades will be 7S, Ace of clubs will be 1C, King of diamonds
+// would be 13D...or KD
+
+ class Card
+{
+public:
+    string faces;
+    int values;
+    Card(int x, string y): values(x), faces(y){}
+
+    void printFace(){ cout << values << " of " << faces <<  endl;}
```

Comment:
Does not follow the printing requirement mentioned in the comments at the beginning

Reply:
Comments lie ... 

But in this case, I am on the side of the code. This is more clear than trying to parse what e.g., 1C or 13D would mean

Reply:
But maybe this has been a requirement for the project. You cant just change what the requirement asks you.

Reply:
the second comment was my own comment...not what was asked by Jean...

Reply:
method should be declared const since it's an inspector method (doesn't change the value) and thus we can use it for const instances of the class